src/noted.cpp

```diff
@@ -0,0 +1,82 @@
+#include <argp.h>
+#include <iostream>
+#include <libnotify/notify.h>
+#include "../include/note.hpp"
+#include <stdlib.h>
```

Comment:
there is already <cstdlib> included in note.hpp

Reply:
cstdlib*

Reply:
Fixed, thanks.